packages/editor-ui/src/plugins/codemirror/completions/constants.ts

```diff
@@ -296,7 +296,7 @@ export const STRING_RECOMMENDED_OPTIONS = [
 	'length',
 ];
 
-export const LUXON_RECOMMENDED_OPTIONS = ['format()', 'minus()', 'plus()', 'diff()', 'extract()'];
+export const LUXON_RECOMMENDED_OPTIONS = ['format()', 'minus()', 'plus()', 'diffTo()', 'extract()'];
```

Comment:
Strictly speaking maybe not in scope but I noticed some methods like `isBetween()` [don't fit](https://share.cleanshot.com/C1yH0KVF) the `Component` category.

Reply:
Move to `compare`